### ISSUE TITLE
Issue 1

### ISSUE THREAD

**Issue by weiji14** [developmentseed/chabud2023#1]
## Background

Change detection for Burned area Delineation (ChaBuD) ECML/PKDD 2023 challenge

- Main website: https://2023.ecmlpkdd.org/submissions/discovery-challenge/challenges
- HuggingFace Spaces: https://2023.ecmlpkdd.org/submissions/discovery-challenge/challenges/
- Datasets:
  - https://huggingface.co/datasets/chabud-team/chabud-ecml-pkdd2023/tree/main
  - https://huggingface.co/datasets/chabud-team/chabud-extra/tree/main

## Goals

- Open source repo showcasing how to perform change detection of burned areas from optical imagery
- Get a blog post (or series of blog posts) out on the data pipeline, machine learning model and results

## Partner and Stakeholders

Development undertaken by Gaia pod @developmentseed, but open to contributions from other interested parties!

## Team

* OD @weiji14
* D @srmsoumya
* D ???

## Milestones

Timeline from https://huggingface.co/spaces/competitions/ChaBuD-ECML-PKDD2023:

- April 11, 2023: competition starts. Train and validation sets are released
- May 8, 2023: Submission opens! Public leaderboard is made available
- June 9, 2023: hidden test set is released (input data only). Private leaderboard is made available.
- **June 18, 2023**: end of the challenge
- June 25, 2023: final leaderboard is released
- July 15, 2023: deadline for final report submission
- July 31, 2023: final decision and winners are announced

## Project Organization

- GitHub Repo: https://github.com/developmentseed/chabud2023
- HuggingFace Spaces: ???

## Documents

- Registration at https://forms.gle/oFXcJHnNaVKnvKak8

**Comment by weiji14** [comment]
## Resources

- TinyCD
  -  Codegoni, A., Lombardi, G., & Ferrari, A. (2022). TINYCD: A (Not So) Deep Learning Model For Change Detection (arXiv:2207.13159). arXiv. https://doi.org/10.48550/arXiv.2207.13159
  - Code: https://github.com/AndreaCodegoni/Tiny_model_4_CD
- DS-UNet
  - Monaco, S., Greco, S., Farasin, A., Colomba, L., Apiletti, D., Garza, P., Cerquitelli, T., & Baralis, E. (2021). Attention to Fires: Multi-Channel Deep Learning Models for Wildfire Severity Prediction. Applied Sciences, 11(22), 11060. https://doi.org/10.3390/app112211060
  - Code: https://github.com/dbdmg/rescue
